pyro/infer/autoguide/initialization.py

```diff
@@ -194,4 +234,8 @@ def _pyro_sample(self, msg):
                     "{} provided invalid shape for site {}:\nexpected {}\nactual {}"
                     .format(self.init_fn, msg["name"], msg["value"].shape, value.shape))
         msg["value"] = value
-        msg["done"] = True
```

Comment:
@eb8680 can you confirm this is ok?  I think we want to avoid setting `done` here so as to enable subsequent reparametrization.